curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/6140c7e645d8e905819f1dd4.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-1
 
 # --description--
 
-Begin with the standard boilerplate. Add your `DOCTYPE` declaration, your `html` element, your `head` and `body` elements.
+Begin with the standard boilerplate. Add your `DOCTYPE` declaration, your `html` element, your `head` and `body` elements. Give the `html` element a `lang` attribute with `en` as its value.
```

Comment:
Thanks @Sboonny , I agree with not to mention the exact attribute, but I suggest clarifying to set it to English because people will be reading this instruction in various languages. This came to my mind because there was a translation contributor trying to recommend the value "ja" for Japanese. (which will not pass the tests)

Reply:
I would have pushed for lenient approach, and allow every 2 letters.

> /<html\s+lang\s*=\s*('|")..\1\s*>/gi

But the campers are creating a page with English as its main language, I am convinced